posydon/interpolation/profile_interpolation.py

```diff
@@ -192,60 +208,86 @@ def load_profiles(self,filename,valid_split=0.2):
         self.initial = np.log10(linear_initial)[binaries[split:]]
         self.scalars = self.scalars.iloc[binaries[split:]]
 
-    def train(self,IF_interpolator,density_epochs=3000,density_patience=200,
-             comp_bounds_epochs=500,comp_bounds_patience=50,loss_history=False,hms_s2=False):
+    def train(self,IF_interpolator,density=True,comp=True,density_epochs=1000,density_patience=200,
+             comp_bounds_epochs=500,comp_bounds_patience=50,loss_history=False,hms_s2=False,
+             depth=12,width=256,depthn=12,widthn=256,lr=0.0001):
         """Trains models for density, H mass fraction, and He mass fraction profile models. 
         Args:
             IF_interpolator (str) : path to '.pkl' file for IF interpolator.
+            density (Boolean) : option to train Density model
+            comp (Boolean) : option to train Composition model
             density_epochs (int) : number of epochs used to train density profile model
             density_patience (int) : patience parameter for NN callback in density profile model
             comp_bounds_epochs (int) : number of epochs used to train composition profiles model
             comp_bounds_patience (int) : patience parameter for NN callback in composition profiles model
             loss_history (Boolean) : option to return training and validation loss histories
             hms_s2 (Boolean) : option to do profiles of star 2 in HMS-HMS grid
+            depth (int) : depth of neural network for principal component weights
+            width (int) : width of neural network for principal component weights
+            depthn (int) : depth of neural network for normalizing value
+            widthn (int) : width of neural network for normalizing value
+            lr (float) : learning rate for neural network training
```

Comment:
Try to avoid that short variable names like `lr`, may just call it `learning_rate`.

Reply:
done!